### DISCUSSION TITLE
PDF viewer

### DISCUSSION THREAD

**Discussion by responsibilities5** [community/community#14446] [Mobile] [answered]
Hi, I have some pdf documents on my repository and I use them for reference every one and I was wondering if there was an option on the GitHub mobile app to view these pdf files?

**Comment by iskae** [comment, marked as answer]
Hi @responsibilities5, thank you for your feedback 🎉 

In its current state the GitHub Mobile Apps should be able to render PDF files when they are part of the repository 👍  We have a 10mb limit for any type of file though. Was the PDF file that you were referencing over 10mbs? 

Here is a small recording of PDF rendering 😊 

https://user-images.githubusercontent.com/26570736/162148022-b7bd706f-c232-48f3-9010-f1e9f0df2166.mp4

**Reply by responsibilities5** [reply]
Hi, thank you for the reply. I checked the size of my file on my computer and it was 4.3 MB on disk. I tried pushing some other pdf's and I was able to view them. I don't know why the other document can't be viewed but I am ok with it. I'll try getting a hard copy. Thank you very much for the support.

**Reply by user1man** [reply]
I have found a link and a repository where I am unable to open the PDF, and the link cannot be opened on a mobile client. The only solution is to open the current page in Safari and then open the link next to it. The page with the link is:
[https://github.com/UberGuidoZ/Flipper#if-youre-new-to-github-in-general-check-out-this-great-guide-from-wr3nch](https://github.com/UberGuidoZ/Flipper#if-youre-new-to-github-in-general-check-out-this-great-guide-from-wr3nch)
[Pdf link](https://github.com/wrenchathome/flipperfiles/blob/main/_Guides/How2Flipper.pdf)
